Added options to the User.info method

chicohernando opened the pull request:
Without it the option of 'compact' cannot be used.

chicohernando committed:
Added options param to the User.info method.  Without it options like 'compact' cannot be used.

chicohernando committed:
Added a missing word to the comment

disengaged commented:
- 1 on this pull request from me. There are a few other methods also missing the options argument.

cmar commented:
Thanks for the pull request. Could you create  a test in https://github.com/cmar/untappd/blob/master/spec/user_spec.rb to make sure the options are working correctly?

chicohernando committed:
Updated README.md to reflect that User.info now supports the options param.
Added a test to verify that passing compact with true removes the recent_brews, media, and checkin fields from the response.

chicohernando commented:
@cmar - Added test to verify that the `User.info` options are working correctly.  Also updated the README.md to reflect that the `options` param is now available.

chicohernando committed:
Updated test to use a be_nil matcher instead of using == nil

chicohernando commented:
@cmar - Are there any more updates that you would like me to make to get this merged?
